kernel/src/hil/aes.rs

```diff
@@ -0,0 +1,14 @@
+use returncode::ReturnCode;
+
+pub trait AESDriver {
```

Comment:
This can be probably stand to be more general. If nothing else we can call the trait something like `Encryption`.